core/common/package.json

```diff
@@ -7,8 +7,8 @@
   "typings": "lib/cjs/core-common",
   "license": "MIT",
   "scripts": {
-    "build": "npm run -s build:cjs",
-    "build:ci": "npm run -s build && npm run -s build:esm",
+    "build": "npm run -s build:cjs && npm run -s build:esm",
```

Comment:
Should we be doing this for all of the packages again? Seems like we're selectively adding some of these back into the main build process.

Reply:
I added this in my webpack5-continued pr in 3.3; this is to get around developers having to manually build the esm version of core-common which is needed when they go to build their test apps.

should be unecessary once steves pr to decouple core-common with the rpc pr goes through